volume/local/local.go

```diff
@@ -363,6 +365,17 @@ func (v *localVolume) unmount() error {
 	return nil
 }
 
+func (v *localVolume) CreatedAt() (time.Time, error) {
+	fileInfo, err := os.Stat(v.path)
```

Comment:
How come we don't add this to the volume store and store it in the db like labels, etc?